resources/js/src/app/components/itemList/filter/FindologicItemFilter.vue

```diff
@@ -57,7 +61,7 @@
               :id="'option-' + value.id"
               class="form-check-input hidden-xs-up"
               type="checkbox"
-              :checked="value.selected"
+              :checked="isSelected"
```

Comment:
Where is `isSelected` defined? I can only find usages.

Reply:
Good catch, not necessary anymore to handle cat filter in ItemFilter, we have ItemCategoryDropdown.